src/MICore/LaunchOptions.xsd

```diff
@@ -371,6 +371,102 @@
+            <xs:annotation>
+              <xs:documentation>Local path to the MI Debugger executable.</xs:documentation>
+            </xs:annotation>
+          </xs:attribute>
+    <xs:attribute name="ExePath" type="xs:string" use="required">
+      <xs:annotation>
+        <xs:documentation>Path to the executable file. This is a path on the Windows machine.</xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+  </xs:complexType>
+
+  <xs:complexType name="AttachOptionsForConnection">
+      <xs:sequence>
+        <xs:element name="SetupCommands" minOccurs="0" maxOccurs="1" type="CommandList">
+          <xs:annotation>
+            <xs:documentation>One or more GDB/LLDB/CLRDBG commands to execute in order to setup the underlying debugger. These are not required.</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="SourceMap" type="SourceMapList" minOccurs="0" maxOccurs="1">
+        </xs:element>
+        <xs:element name="ServerOptions" type="ServerOptions" minOccurs="0" maxOccurs="1">
+          <xs:annotation>
+            <xs:documentation>If specified then attach using client-server mode.</xs:documentation>
+          </xs:annotation>
+        </xs:element>
+      </xs:sequence>
+      <xs:attribute name="AdditionalSOLibSearchPath" type="xs:string" use="optional">
+        <xs:annotation>
+          <xs:documentation>Semicolon separated list of directories to use to search for .so files. Example: 'c:\dir1;c:\dir2'</xs:documentation>
+        </xs:annotation>
+      </xs:attribute>
+      <xs:attribute name="MIMode" type="MIMode" use="required"/>
+    <xs:attribute name="WorkingDirectory" type="xs:string" use="optional">
+      <xs:annotation>
+        <xs:documentation>Working directory on the remote device if remote debugging, on the local device if client-server debugging.</xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+      <xs:attribute name="DebugChildProcesses" type="xs:boolean" use="optional">
+        <xs:annotation>
+          <xs:documentation>
+            If true, automatically attach debugger to any child processes created.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:attribute>
+      <xs:attribute name="VisualizerFile" type="xs:string" use="optional">
+        <xs:annotation>
+          <xs:documentation>.natvis file to be used when debugging this platform</xs:documentation>
+        </xs:annotation>
+      </xs:attribute>
+      <xs:attribute name="ShowDisplayString" type="xs:boolean" use="optional">
+        <xs:annotation>
+          <xs:documentation>
+            If true, instead of showing Natvis-DisplayString value as a child of a dummy element, it is shown immediately.
+            Should only be enabled if debugger is fast enough providing the value.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:attribute>
+    <xs:attribute name="ConnectionName" type="xs:string" use="optional">
```

Comment:
>xs [](start = 5, length = 2)

nit: The indenting seems a bit off in parts of this. Might want to reformat it all to make it easier to read.

Reply:
fixed